libs/output-mapping/src/Mapping/MappingFromConfigurationSchemaColumnDataType.php

```diff
@@ -12,7 +12,46 @@ public function __construct(private readonly array $mapping)
     {
     }
 
-    public function getBaseType(): string
+    public function getTypeName(?string $backend = null): string
+    {
+        if ($backend === null) {
+            return $this->getBaseTypeName();
+        }
+
+        try {
+            return $this->getBackendTypeName($backend);
+        } catch (InvalidOutputException $e) {
+            return $this->getBaseTypeName();
+        }
+    }
+
+    public function getLength(?string $backend = null): ?string
+    {
+        if ($backend === null) {
+            return $this->getBaseLength();
+        }
+
+        try {
+            return $this->getBackendLength($backend);
+        } catch (InvalidOutputException $e) {
+            return $this->getBaseLength();
+        }
+    }
+
+    public function getDefaultValue(?string $backend = null): ?string
+    {
+        if ($backend === null) {
+            return $this->getBaseDefaultValue();
+        }
+
+        try {
+            return $this->getBackendDefaultValue($backend);
+        } catch (InvalidOutputException $e) {
+            return $this->getBaseDefaultValue();
+        }
+    }
+
```

Comment:
Pro tyhle metody bys měl doplnit i testy na tan fallback při `InvalidOutputException`. Že to vrací ty base hodnoty.

Reply:
to je pokrytý tímhle testem
https://github.com/keboola/platform-libraries/pull/268/files#diff-d14a63c54f8e633e5a9e7a24e7262126daa4d8ebfd06e06f4cfc37f6542ad7dbR125-R126

1/ snowflake existuje tak vrátí snflk hodnotu
2/ bigquery neexistuje tak vrátí base hodnotu

Reply:
Díky, to jsem přehlídl že to tam je jen pro jeden backend :)